spec/notifications/new_aircraft_created_notification_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe NewAircraftCreatedNotification, type: :model do
+
+  context 'when aircraft are created' do
+    it 'creates notifications' do
+
+      # Must eagerly create a user first, or notifications will not be created
+      user = FactoryBot.create(:user)
+      aircraft = FactoryBot.create(:aircraft)
+
+      # TODO: instead of count, very actual contents of notifications
+      # Note also, notifications temporarily set to User.all
+
+      expect(aircraft).to be_valid
```

Comment:
rather than having these 3 different expect statements, we could try
`expect { FactoryBot.create(:aircraft) }.to change { Notification.count }.by(1)`.

Something along those lines (checks that creating an aircraft increases the Notification count by 1)